common/sagas/schedule/currentScheduleTimestamp.ts

```diff
@@ -0,0 +1,24 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { SagaIterator } from 'redux-saga';
+import { TypeKeys } from 'actions/schedule/constants';
+import { SetScheduleTimestampFieldAction, setScheduleTimestampField } from 'actions/schedule';
+import { SetCurrentScheduleTimestampAction } from 'actions/schedule/actionTypes/scheduleTimestamp';
+
+export function* setCurrentScheduleTimestamp({
+  payload: raw
+}: SetCurrentScheduleTimestampAction): SagaIterator {
+  let value: Date | null = null;
+
+  value = new Date(raw);
+
+  yield call(setField, { value, raw });
+}
+
+export function* setField(payload: SetScheduleTimestampFieldAction['payload']) {
```

Comment:
This function isn't used anywhere externally, would rather just have inlineyield put(setScheduleTimestampField(payload)); in the two references above to avoid extra indirection